simple-installer/setup_monster-ui

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+
+set -e
+set -o nounset
+
+lockfile="/var/lock/subsys/setup_kazoo-ui.lockfile"
+. /opt/kazoo_install/setup_common
+
+trap clean_exit SIGHUP SIGTERM ERR EXIT
+trap int_exit SIGINT SIGQUIT
+
+kazoo_ui_conf_js="/var/www/html/monster-ui/js/config.js"
+
+#KAZOO-UI URL
+url=""
+url_dbg="setting kazoo-ui URL address for accessing API - TOKEN"
```

Comment:
s|kazoo-ui|monster-ui|g

There are some lines in this file to display it is operation on kazoo-ui.  This will be confusing and should be corrected ;)